src/QueryVisualizationConnection.ts

```diff
@@ -2,7 +2,7 @@
 
  MIT License
 
- Copyright (c) 2022 Looker Data Sciences, Inc.
+ Copyright (c) 2024 Looker Data Sciences, Inc.
```

Comment:
this is a confused rename on this file name, but ok

Reply:
This should probably be 2025 but I'm going to leave it